src/pedal_handler.cpp

```diff
@@ -161,12 +190,12 @@ void PedalHandler::verify_pedals(
         // mcu_status.set_no_accel_implausability(true);
         accel_is_plausible = true;
     }
-    Serial.printf("Torque 1: %d Torque 2: %d Torque Sum: %f Torque Average %f Torque Difference: %f\n",torque1,torque2,torqSum,torqAvg,torqDiff);
+    Serial.printf("Torque 1: %d Torque 2: %d Torque Sum: %f Torque Average %f Torque Difference: %f\n", torque1, torque2, torqSum, torqAvg, torqDiff);
```

Comment:
these need to be in a struct or something so we can send over CAN to monitor remotely